Ontology/Asset/Component/PipeConnection.json

```diff
@@ -0,0 +1,29 @@
+{
+  "@id": "dtmi:digitaltwins:rec_3_3:asset:PipeConnection;1",
+  "@type": "Interface",
+  "contents": [
+    {
+      "@type": "Property",
+      "displayName": {
+        "en": "diameter"
+      },
+      "name": "diameter",
```

Comment:
I propose calling this outsideDiameter for consistency w/ insideDiameter.

Reply:
Let me look into this. I think pipes generally get specified with a nominal diameter or "pipe size" and then have an outside diameter property. This is what I've seen on most product data sheets. I'll come back with a recommendation for an update.

Reply:
Ok, I have come to a recommendation based on researching North America and ISO standards and specifications as well as other modeling. In order to keep the terms flexible for international use, I'm recommending we have 3 properties on **PipeConnection**:

* nominalDiameter - the standard or non-specific diameter which correlates to a Nominal Pipe Size (North America) or Nominal Diameter (ISO 6708)
* insideDiameter - the actual diameter measured from the inner walls of the pipe
* outsideDiameter - the actual diameter measured from the outer walls of the pipe

Reply:
@hammar - I forgot to tag you above in my above proposal regarding my proposal for diameter properties.